storage/backend/boltoption_unix.go

```diff
@@ -30,5 +30,5 @@ import (
 // silently ignore this flag. Please update your kernel to prevent this.
 var boltOpenOptions = &bolt.Options{
 	MmapFlags:       syscall.MAP_POPULATE,
-	InitialMmapSize: InitialMmapSize,
+	InitialMmapSize: int(InitialMmapSize),
```

Comment:
This will make `etcd` unreliable for `amd64`, and was the cause to the https://github.com/coreos/etcd/pull/4715#issuecomment-193884177 failure. If you merge this `etcd` won't work. See https://play.golang.org/p/3Slqmtf9JC